frappe/database/postgres/database.py

```diff
@@ -73,13 +73,22 @@ def get_connection(self):
 
 		return conn
 
-	def escape(self, s, percent=True):
+	def escape(self, s, percent=True, underscore=False):
```

Comment:
I think the docstring could be fixed too ;)

Reply:
👍

Reply:
Docstring updated 👍